### ISSUE TITLE
Solidus 4.0 compatibility

### ISSUE THREAD

**Issue by pelargir** [AlchemyCMS/alchemy-solidus#92]
We need a release of this gem compatible with Solidus 4.0

**Comment by tvdeyen** [comment]
@pelargir please try #93 in your Gemfile

**Comment by pelargir** [comment]
@tvdeyen I'm getting this when I bundle. Can you relax the solidus requirement back to 6?

```
Bundler could not find compatible versions for gem "alchemy_cms":
  In snapshot (Gemfile.lock):
    alchemy_cms (= 6.1.5)

  In Gemfile:
    alchemy_cms (~> 6.1.1)

    alchemy-bugsnag (~> 1.0.0) was resolved to 1.0.0, which depends on
      alchemy_cms (>= 4.6.7, < 7)

    alchemy-dragonfly-s3 (~> 6.0.0) was resolved to 6.0.1, which depends on
      alchemy_cms (>= 6.0.0, < 7)

    alchemy-solidus was resolved to 6.0.0, which depends on
      alchemy_cms (>= 7.0.0.a, < 8)

Running `bundle update` will rebuild your snapshot from scratch, using only
the gems in your Gemfile, which may resolve the conflict.
```

**Comment by tvdeyen** [comment]
@pelargir unfortunately it is not that easy, because we removed Alchemy 6 support in #90 

Will try backport the Solidus 4 compatibility into a version that can support both.
